azure-pipelines.yml

```diff
@@ -2,7 +2,11 @@
 # https://aka.ms/yaml
 
 trigger:
-  - master
+  batch: true
+  branches:
+    include:
+    - master
```

Comment:
isn't this the default?

Reply:
this is to restrict the pipeline run to one run only so that we don't get issues for tfstate locking

Reply:
i mean the branch.include.master, batch: true looks nice